SU2_CFD/src/drivers/CDriver.cpp

```diff
@@ -2490,14 +2490,23 @@ void CDriver::InitializeInterface(CConfig **config, CSolver***** solver, CGeomet
           if (rank == MASTER_NODE) cout << "boundary displacements from the structural solver." << endl;
         }
         else if (fluid_donor && fluid_target) {
-                /*--- Mixing plane for turbo machinery applications. ---*/
-          if (config[donor]->GetBoolMixingPlaneInterface()) {
-            interface_type = MIXING_PLANE;
-            auto nVar = solver[donor][INST_0][MESH_0][FLOW_SOL]->GetnVar();
-            interface[donor][target] = new CMixingPlaneInterface(nVar, 0);
-            if (rank == MASTER_NODE) {
-              cout << "Set mixing-plane interface from donor zone "
-                  << donor << " to target zone " << target << "." << endl;
+          /*--- Interface handling for turbomachinery applications. ---*/
+          if (config[donor]->GetBoolTurbomachinery()) {
+            auto interfaceIndex = donor+target; // Here we assume that the interfaces at each side are the same kind
```

Comment:
Out of curiosity, why is this computation of an interface index not ambiguous? Like, donor 1 and target 4 would yield the same as donor 2 and target 3?

Reply:
Because we are making a strong assumption that MARKER_TURBOMACHINERY follows the main flow path of the machine, this is done throughout other parts of the code too. These parts as well as this index will need to be refactored to include secondary flow paths in the future.

Reply:
I see :+1: